### PR TITLE
lda user stat updater

### PR DESCRIPTION
**Pull request by yingjieMiao**
@stkem

### CODE REVIEW

File: kifi-backend/modules/cortex/app/com/keepit/cortex/utils/MatrixUtils.scala [hunk 1]
```diff
@@ -92,4 +92,12 @@ object MatrixUtils {
     }
     (tuples.map { _._1 }.toArray, tuples.map { _._2 }.toArray)
   }
+
+  def MDistanceDiagGaussian(sample: Array[Double], mean: Array[Double], variance: Array[Double]) = {
```
Comment (stkem) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/utils/MatrixUtils.scala:96]: I recommend going old school and using while loops and such in this function. For numerical things like this that can easily be an order of magnitude faster.

**Comment by stkem** [comment]
made one comment. lgtm otherwise

File: kifi-backend/modules/cortex/app/com/keepit/cortex/utils/MatrixUtils.scala (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/utils/MatrixUtils.scala:96]: agree. I will send a PR later to optimize this in a few other places.

Comment (stkem) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/utils/MatrixUtils.scala:96]: soon please. Should be in prod before we restart the computation.

**Pull request merged**